telco-hub/configuration/reference-crs/optional/cert-manager/certManagerClusterIssuer.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: selfsigned-issuer
+spec:
+  selfSigned: {}
```

Comment:
For the reference configuration we should use an ACME ClusterIssuer. Adjust the config and name here and where referenced below.

Reply:
Done.  Switched it to the ACME example from the documentation.